centreon/packages/ui/src/ActionsList/index.tsx

```diff
@@ -28,12 +29,16 @@ const useStyles = makeStyles()({
   }
 });
 
-const ActionsList = ({ className, actions }: Props): JSX.Element => {
+const ActionsList = ({
+  className,
+  actions,
+  onClick: onMenuListClick
+}: Props): JSX.Element => {
   const { cx, classes } = useStyles();
   const { t } = useTranslation();
 
   return (
-    <MenuList className={cx(classes.list, className)}>
+    <MenuList className={cx(classes.list, className)} onClick={onMenuListClick}>
```

Comment:
I would change the naming the other way around: let onClick be in this scope, and name onClick: onActionClick below